File: features/wx/wx/FormId.go
package wx

import (
	"github.com/hailongz/golang/db"
)

type FormId struct {
	db.Object
	Appid  string `json:"appid" name:"appid" title:"appid" length:"64" index:"ASC"`
	Openid string `json:"openid" name:"openid" title:"openid" length:"128" index:"ASC"`
	Formid string `json:"formid" name:"formid" title:"formid" length:"128"`
	Etime  int64  `json:"etime" name:"etime" title:"过期时间" index:"DESC"`
}

func (O *FormId) GetName() string {
	return "formid"
}

func (O *FormId) GetTitle() string {
	return "FormId"
}

File: features/wx/ts/FormId.ts
import { int64 } from "./lib/less";
import { UserType } from './UserType';

/**
 * FormId
 * @type db
 */
export class FormId {

    /**
     * ID
     */
    id: int64 = 0

    /**
     * appid
     * @index ASC
     * @length 64
     */
    appid: string = ''

    /**
     * openid
     * @index ASC
     * @length 128
     */
    openid: string = ''

    /**
     * formid
     * @length 128
     */
    formid: string = ''

    /**
     * 过期时间
     * @index DESC
     */
    etime: int64 = 0

}